tests_v2/test_tmux_compat_show_options.py

```
@@ -0,0 +1,255 @@
+        raise cmuxError("Could not locate cmux CLI binary; set CMUXTERM_CLI")
+    candidates.sort(key=lambda p: os.path.getmtime(p), reverse=True)
+    return candidates[0]
+
+
+def _run_tmux_compat(cli: str, args: List[str]) -> subprocess.CompletedProcess:
+    env = dict(os.environ)
+    env.pop("CMUX_WORKSPACE_ID", None)
+    env.pop("CMUX_SURFACE_ID", None)
+    env["CMUX_SOCKET_PATH"] = SOCKET_PATH
+    cmd = [cli, "--socket", SOCKET_PATH, "__tmux-compat"] + args
+    return subprocess.run(cmd, capture_output=True, text=True, check=False, env=env)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)